src/components/Autosuggest/Autosuggest.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import Autosuggest from "react-autosuggest";
+import Label from "../Label/Label";
+import Input from "../Input/Input";
+import { InputTypes } from "../Input/InputTypes";
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+
+const AutosuggestComp = ({
+    defaultValue,
+    childRef,
+    list,
```

Comment:
is list going to be ok with just being a list of strings?  We won't ever want an autosuggest list with half-bolded elements, etc?

Reply:
It can be anything. I didn't write an interface for this though it probably would've been something like `:any`.